shared/actions/teams.tsx

```diff
@@ -1505,6 +1505,34 @@ const startAddMembersWizard = (_: TeamsGen.StartAddMembersWizardPayload) =>
   RouteTreeGen.createNavigateAppend({
     path: ['teamAddToTeamFromWhere'],
   })
+const finishNewTeamWizard = async (state: TypedState) => {
+  const {name, description, open, openTeamJoinRole, showcase, addYourself} = state.teams.newTeamWizard
+  const {avatarFilename, avatarCrop, channels, subteams} = state.teams.newTeamWizard
+  const teamInfo: RPCTypes.TeamCreateFancyInfo = {
+    avatar: avatarFilename ? {avatarFilename, crop: avatarCrop?.crop} : null,
+    chatChannels: channels,
+    description,
+    joinSubteam: addYourself,
+    name,
+    openSettings: {joinAs: RPCTypes.TeamRole[openTeamJoinRole], open},
+    showcase,
+    subteams,
+    users: state.teams.addMembersWizard.addingMembers.map(member => ({
+      assertion: member.assertion,
+      role: RPCTypes.TeamRole[member.role],
+    })),
+  }
+  try {
+    const teamID = await RPCTypes.teamsTeamCreateFancyRpcPromise({teamInfo}, Constants.teamCreationWaitingKey)
+    return [
+      TeamsGen.createFinishedNewTeamWizard(),
+      TeamsGen.createFinishedAddMembersWizard(),
```

Comment:
prefer to remove this and have the store changes / nav be a side effect of `finishedNewTeamWizard`